PhpRbac/src/PhpRbac/core/lib/rbac.php

```diff
@@ -775,6 +775,27 @@ function roles($Permission, $OnlyIDs = true)
     		    WHERE PermissionID=? ORDER BY TP.ID", $Permission );
 		}
 	}
+	/**
+	 * Returns all users with a permission
+	 *
+	 * @param integer $Permission
+	 *        	ID
+	 * @return Array 1D or null
+	 */
+	function users($Permission)
+	{
+		if (!is_numeric($Permission)) $Permission = $this->returnId($Permission);
```

Comment:
```
if (!is_int($PermissionId)) {
    throw new InvalidArgumentException("`$PermissionId` should be type of integer")
}
```

Reply:
agree. this check can be performed after permission string is resolved into id